Dockerfile

```diff
@@ -1,3 +1,10 @@
-FROM starefossen/github-pages:onbuild
+# Build the docs
+FROM starefossen/github-pages:147 AS build_docs
+COPY . /source
+RUN jekyll build --source /source --destination /site
 
-CMD bundle update && bundle exec jekyll serve -d /_site --watch -H 0.0.0.0 -P 4000
+# Reset with nginx
+FROM nginx:alpine
+COPY nginx.conf /etc/nginx/nginx.conf
+COPY --from=build_docs /site /usr/src/app/target
```

Comment:
If we copy to `/usr/share/nginx/html`, there's even less customisation needed in the default configuration 👍